src/coord/calendar/Calendar.js

```diff
@@ -361,6 +361,7 @@ Calendar.prototype = {
      * @return {Object}       obj
```

Comment:
If you wish to use show the structure of `range`, please do it here in the form of [jsdoc](https://jsdoc.app/tags-param.html).